components/size_aware_image.jsx

```diff
@@ -181,6 +179,8 @@ export default class SizeAwareImage extends React.PureComponent {
             ariaLabelImage += ` ${fileInfo.name}`.toLowerCase();
         }
 
+        const svgStyleAttribute = fileInfo.extension === FileTypes.SVG && {style: {width: this.state.imageWidth}};
```

Comment:
Similarly, the `fileInfo` is also optional, and it'll likely be undefined in those cases. I got an error here when I made a post while the post shown in the channel was still "fake"

Reply:
Fix